Packs/GigamonThreatINSIGHT/Integrations/GigamonThreatINSIGHT/GigamonThreatINSIGHT.yml

```
@@ -0,0 +1,900 @@
+commonfields:
+  id: Gigamon ThreatINSIGHT
+  version: -1
+name: Gigamon ThreatINSIGHT
+display: Gigamon ThreatINSIGHT
+category: Network Security
+description: Gigamon ThreatINSIGHT is a cloud-native network detection and response solution built for the rapid detection of threat activity, investigation of suspicious behavior, proactive hunting for potential risks, and directing a fast and effective response to active threats.
+configuration:
+- display: API Token
+  name: api_key
+  type: 4
+  required: true
+- display: First Fetch Time (Amount of day before current date)
```

```suggestion
- display: First fetch timestamp (<number> <time unit>, e.g., 12 hours, 7 days)
```
It's not necessarily days, let's add this.